common/math.go

```diff
@@ -0,0 +1,45 @@
+package common
```

Comment:
I think the package name is not convenient for other repo to use, since `common` may be conflict with their own packages.

Instead, how about rename to `mathutil`? Generally, this repo contains many different utilities with package name `xxxutil`.

Reply:
moved to math package